pkg/jobworker/cgroup_test.go

```diff
@@ -0,0 +1,94 @@
+package jobworker
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+const testName = "TEST"
+
+var testOpts = JobOpts{100, 50, 100 * CgroupMB}
+
+// exists returns whether the given file or directory exists
+func exists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return false, err
+}
+
+func TestCgroupController(t *testing.T) {
+	testDir := fmt.Sprintf("/tmp/%s", testName)
```

Comment:
[`T.TempDir()`](https://pkg.go.dev/testing#T.TempDir) is your friend here:

```suggestion
	testDir := t.TempDir()
```

... and the cleanup code below